yarn.lock

```diff
@@ -1725,6 +1725,14 @@
     shuffle-seed "^1.1.6"
     titleize "^1.0.1"
 
+"@ovh-ux/manager-server-sidebar@^0.4.2":
```

Comment:
The version in the repository is `0.5.0`. Should not be in yarn.lock.

Reply:
Please rebase on `develop` (changes done in https://github.com/ovh-ux/manager/commit/8d5d59e25b5119d8d53d19a9b5907f746699d161)

Reply:
done, thx